test/extended/openstack/volumes.go

```diff
@@ -280,6 +352,17 @@ func getMonitoringPvcs(ctx context.Context, dc dynamic.Interface) ([]objx.Map, e
 	return objects(objx.Map(obj.UnstructuredContent()).Get("items")), nil
 }
```

Comment:
This appears to be dead code now. Can we remove it?

Reply:
Ack.  Thanks for noticing!